wetmap/src/components/searchTool/view.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import screenData from '../newModals/screenData.json';
+// import style from './style.module.scss';
+import TextInputField from '../newModals/textInput';
+import ListItemView from './searchToolListItem/view';
+
+import config from '../../icons/_config.json';
+
+type IconName = keyof typeof config;
+
+type SearchOption = {
+  id:     number
+  title:  string
+  source: IconName
+};
+
+type SearchViewProps = {
+  handleClear:    () => void
+  handleChange:   (text: string) => void
+  handleSelect:   (text: string) => void
+  inputText:      string
+  options:        SearchOption[]
+  setSearchValue: React.SetStateAction<string>
+};
+
+export default function SearchView(props: SearchViewProps) {
+  return (
+    <div className="mt-6 mr-4 pr-4 full-height" style={{ overflow: 'scroll' }}>
```

Comment:
For this specific div, i see when running the site locally that it extends past the visible parent modalContainer. As a result when scrolling to the bottom of the search results the last result will be partially cut off
<img width="692" alt="image" src="https://github.com/user-attachments/assets/d3df4ebf-ebb7-4803-ad5e-85eefe39348b">

I've scrolled to the very bottom in this image. You can see what i mean when the last result is cutoff
<img width="638" alt="image" src="https://github.com/user-attachments/assets/5a408790-83d8-4914-afd0-8093c50b77c1">

Reply:
Perhaps this can be remedied by ensuring the component is at maximum 100% height of its parent?

Reply:
I added a margin bottom to sort it out, the last option is now fully on screen with a bit of space to spare, even though once this moves into the top bar the options should actually float above the bar and cascade down over the map as needed.